profiles/factories.py

```diff
@@ -43,10 +44,21 @@ class ProfileFactory(DjangoModelFactory):
     edx_bio = FuzzyText()
     about_me = FuzzyText()
 
+    romanized_first_name = LazyAttribute(lambda x: FAKE.first_name())
+    romanized_last_name = LazyAttribute(lambda x: FAKE.last_name())
+
+    address1 = LazyAttribute(lambda x: '{} {}'.format(FAKE.building_number(), FAKE.street_name()))
+    address2 = LazyAttribute(lambda x: FAKE.secondary_address())
+    address3 = None
+
+    postal_code = LazyAttribute(lambda x: FAKE.postcode())
     city = LazyAttribute(lambda x: FAKE.city())
     country = LazyAttribute(lambda x: FAKE.country_code())
     state_or_territory = LazyAttribute(lambda x: FAKE.state())
 
+    phone_number = LazyAttribute(lambda x: FAKE.numerify('###-###-####'))
+    phone_country_code = FuzzyInteger(0, 999)
```

Comment:
Actually, your comment here made me scrutinize this field a little further. I've changed the type on the model to a `CharField` because it can have leading zeroes so the previous type was inappropriate.